bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/themes/ColorsAndFontsPreferencePage.java

```diff
@@ -331,9 +334,22 @@ private class PresentationLabelProvider extends LabelProvider implements IFontPr
 
 		private int usableImageSize = -1;
 
+		private final Runnable updateControlsAndRefreshTreeRunnable = new Runnable() {
```

Comment:
can't we write this as a lambda as:

```
private final Runnable updateControlsAndRefreshTreeRunnable = () -> {
	if (fontChangeButton == null || fontChangeButton.isDisposed()) {
		return;
	}
	updateControls();
	tree.getViewer().refresh();
};
```

as "listener" below also uses a lambda already this would make the code more consistent with regards to the code style